WarehouseManagement.Api/Controllers/DeliveryController.cs

```diff
@@ -130,4 +130,14 @@ public async Task<IActionResult> AllDeleted()
 
         return Ok(model);
     }
+
+    [HttpGet("history/{id}")]
+    [ProducesResponseType(200, Type = typeof(DeliveryHistoryDto))]
+    [ProducesResponseType(404)]
+    public async Task<IActionResult> GetHistory([FromQuery] int id)
```

Comment:
can you also post the body of how the result looks like when you call the API?

Reply:
Like a screenshot? 😄

Reply:
yes